cmd/validator/validator.go

```diff
@@ -162,6 +168,10 @@ Supported formats: standard, json, junit (default: "standard")`,
 		}
 	}
 
+	if err := buildExcludeFileTypesFromFileTypes(excludeFileTypesPtr, fileTypesPtr); err != nil {
```

Comment:
BTW @kehoecj is there a way to validate the behavior of exclusive exclusion (cannot use filter + exclusion) for types in the tests?

Or the code needs to be refactored a lot?

Reply:
This should be possible to validate in testing.

Reply:
Then I think it should be part of the PR